rfc/20/README.md

```diff
@@ -146,7 +146,8 @@ Exit codes: 0 if no issues found, 1 otherwise.
 | data_iface      | `cluster>{$server-node}>network>data>interfaces` | This is actually a comma-separated list of strings. The first iface will be taken from that list. |
 | io_disks        | `cluster>{$server-node}>storage>data_devices`    |                                                                                                   |
 | data_iface_type | `cluster>{$server-node}>network>data>interface_type` | Data interface type (tcp|o2ib)                                                                |
-| s3_client_count | `cluster>{$server-node}>s3_instances`            | No of s3 server instances                                                                         |
+| name            | `cortx>motr>clients[N]>name`                     | we need to check name of the client is rgw                                                        |
+| instances       | `cortx>motr>clients[N]>num_instances`            | No of rgw server instances, we need to check when name is rgw `(cortx>motr>clients[N]>name)`      |
```

Comment:
Same as above in let's avoid explicitly just mentioning rgw, here and at other places in the document.